data/src/main/java/cmm/apps/esmorga/data/user/model/UserDataModel.kt

```diff
@@ -0,0 +1,7 @@
+package cmm.apps.esmorga.data.user.model
+
+data class UserDataModel(
+    val name: String,
```

Comment:
we agreed to name the variables with a prefix indicating the layer or we just proposed it and never actually implemented it?